packages/zowe-explorer/__tests__/__unit__/commands/TsoCommandHandler.unit.test.ts

```diff
@@ -299,7 +305,16 @@ describe("TsoCommandHandler unit testing", () => {
             value: jest.fn(() => {
                 return {
                     allProfiles: [{ name: "firstName", profile: { user: "firstName", password: "12345" } }, { name: "secondName" }],
-                    defaultProfile: { name: "firstName" },
+                    dgetDefaultProfile: jest.fn(() => ({
```

Comment:
Should this say `getDefaultProfile`, or be removed altogether as it appears to be unused?